Tests/StreamChatTestTools/Extensions/Unique/Date+Unique.swift

```diff
@@ -18,3 +18,13 @@ extension Date {
         Date(timeIntervalSince1970: .random(in: (date.timeIntervalSince1970 + 1)...Date.distantFuture.timeIntervalSince1970))
     }
 }
+
+extension NSDate {
+    static var unique: NSDate {
+        Date.unique as NSDate
+    }
+
+    static func unique(after date: NSDate) -> NSDate {
+        Date.unique(after: date as Date) as NSDate
+    }
+}
```

Comment:
nit: I would go with typealias here so if we decide to remove `DBDate` we get a compilation error in test target and won't forget to remove this leftover
```suggestion
extension DBDate {
    static var unique: DBDate {
        Date.unique.bridgedDate
    }

    static func unique(after date: DBDate) -> DBDate {
        Date.unique(after: date.bridgedDate).bridgedDate
    }
}
```